st2actions/st2actions/runners/ssh/parallel_ssh.py

```diff
@@ -216,6 +218,12 @@ def _execute_in_pool(self, execute_method, **kwargs):
         return results
 
     def _connect(self, host, results, raise_on_any_error=False):
+        bastion_hostname, bastion_port = None, None
+
+        if "!" in host:
```

Comment:
Typically breaking out magic string as a constant helps. 

Probably place it somewhere https://github.com/StackStorm/st2/blob/master/st2common/st2common/constants/runners.py